_search-plugins/neural-text-search.md

```diff
@@ -110,6 +110,9 @@ PUT /my-nlp-index/_doc/2
 
 Before the document is ingested into the index, the ingest pipeline runs the `text_embedding` processor on the document, generating text embeddings for the `passage_text` field. The indexed document includes the `passage_text` field, which contains the original text, and the `passage_embedding` field, which contains the vector embeddings. 
 
+### Ingestion Recommendation
+The throughput for ingestion varies with the bulk size. Neural search plugin supports different remote connectors for inference like SageMaker, Amazon Bedrock, OpenAI etc. We recommend to have bulk size up to 500 docs for SageMaker and for all other connectors we recommend to refer the API limits for bulk size to get the best throughput.
```

Comment:
```suggestion
Ingestion throughput varies with bulk size. Neural search supports different [connectors]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/connectors/) for models hosted on external platforms, such as SageMaker, Amazon Bedrock, and OpenAI. For SageMaker, we recommend bulk size up to 500 documents. For all other connectors, we recommend to refer to the API limits for bulk size for best throughput. For more information, see [Bulk API]({{site.url}}{{site.baseurl}}/api-reference/document-apis/bulk/).
{: .important}

```

Reply:
I don't think we have any limits discussed on the [Bulk API page](https://opensearch.org/docs/latest/api-reference/document-apis/bulk/). Should we add them?